compiler/ilgen/OMRMethodBuilder.hpp

```diff
@@ -28,6 +28,10 @@
 #include "ilgen/IlBuilder.hpp"
 #include "env/TypedAllocator.hpp"
 
+#if defined (_MSC_VER) && _MSC_VER < 1900
```

Comment:
I know it's more files to change, but I would prefer this code go into the .cpp files, which will also mean moving some code out of OMRMethodBuilder.hpp that uses snprintf. I discussed with Charlie and he has offered to do that work as an immediate follow-up PR so that we can get this one merged.